hera_qm/ant_metrics.py

```diff
@@ -0,0 +1,394 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import aipy as a 
+from copy import deepcopy
+from pyuvdata import UVData
+import json
+
+
+#######################################################################
+#Low level functionality that is potentially reusable
+#######################################################################
+
+
+def per_antenna_modified_z_scores(metric):
+    '''For a given metric, stored as a (ant,antpol) dictonary, computes the per-pol modified z-score 
```

Comment:
I use this format for "simple," self-explanatory functions and the Arguments/Returns format for more complicated ones. Let me know if you think we definitely need to change it. 

I personally find that huge comment blocks makes the code less readable because it separates the function definition from the rest of the function.